api/node/__test__/js_value_conversion.spec.mts

```diff
@@ -610,13 +611,10 @@ test("MapModel undefined rowData sourcemodel", (t) => {
     const nameModel: ArrayModel<number> = new ArrayModel([1, 2, 3]);
 
     let mapFunctionCallCount = 0;
-    const mapModel = new private_api.MapModel<number, string>(
-        nameModel,
-        (data) => {
-            mapFunctionCallCount++;
-            return data.toString();
-        },
-    );
+    const mapModel = new MapModel<number, string>(nameModel, (data) => {
```

Comment:
Am I correct in assuming that your plan is to finish the `MapModel` implementation in follow-up changes before the release then?

Reply:
Since we are so close to the release, I'd prefer to first finish MapModel because making it public. Just in case it can't happen in time.

Reply:
Great :)